components/DatePicker/DatePicker.jsx

```diff
@@ -47,37 +47,48 @@ export default class DatePicker extends React.Component<Props> {
   }
 
   state = {
-    open: false,
     valid: true,
   }
 
+  props: Props
```

Comment:
You can drop this props definition if you want. The class already `extends React.Component<Props>`, defining an empty `props: Props` is redundant in this case.